docs/private-networks/concepts/node-sync-private.md

```diff
@@ -19,14 +19,14 @@ To sync Besu on a private network:
 - Set the network ID and bootnodes specific to your private network.
 - Implement permissioning features to control network access.
 
-Choose the appropriate sync mode based on your private network's requirements and node purposes.
+The following is an overview of the private network sync modes:
 
-| Sync mode | Description | Requirements | Limitations |
-|-----------|-------------|--------------|-------------|
-| [Snap](../../public-networks/concepts/node-sync.md#snap-synchronization) | Recommended for fastest sync and lowest storage requirements on Mainnet. Downloads as many leaves of the trie as possible and reconstructs the trie locally. Faster than fast sync. | Available as an _early access feature_ in Besu version 24.7.1 or later | Cannot switch from fast sync to snap sync mid-process. |
+| Sync mode                                                                            | Description | Requirements | Limitations |
+|--------------------------------------------------------------------------------------|-------------|--------------|-------------|
+| [Snap](../../public-networks/concepts/node-sync.md#snap-synchronization)             | Recommended for fastest sync and lowest storage requirements on Mainnet. Downloads as many leaves of the trie as possible and reconstructs the trie locally. Faster than fast sync. | Available as an _early access feature_ in Besu version 24.7.1 or later | Cannot switch from fast sync to snap sync mid-process. |
 | [Checkpoint](../../public-networks/concepts/node-sync.md#checkpoint-synchronization) | Syncs from a specific checkpoint block configured in the genesis file. Fastest sync mode with lowest storage requirements. | Besu version 22.4.3 or later | Not supported for QBFT or IBFT 2.0 networks without a checkpoint configuration. |
-| [Fast](../../public-networks/concepts/node-sync.md#fast-synchronization) |Default for named networks except `dev`. Downloads block headers and transaction receipts, verifies chain from genesis block. | None | Not supported with private transactions. |
```

Comment:
I am not sure this should be removed. I think this was called out by the dev in the previous PR - Fast sync (`SNAP`) is the default for most named networks. 

Full sync is the default for private networks and unnamed networks, so  I think the 'dev' network defaults to full sync

Reply:
> - Fast sync (SNAP) is the default for most named networks.

Fast sync and snap sync are two different syncs. The existing docs state that fast sync is the default for most named networks, which appears to be false according to the options reference, so I removed that from both tables and added on the public networks page that snap sync is the default for named networks.

Reply:
The current docs don't have anything on full sync being the other default, so I can add that in.